hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/impl/TestHddsDispatcher.java

```diff
@@ -132,6 +132,7 @@ public void testContainerCloseActionWhenFull() throws IOException {
 
     } finally {
       volumeSet.shutdown();
+      ContainerMetrics.remove();
```

Comment:
Should we call shutdownDispatcher() here as well instead of just ContainerMetrics.remove()?

Reply:
I removed that method and unify all the calls to ContainerMetrics.remove()